cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -63,36 +63,36 @@ extends ColumnStore with CassandraChunkSource with StrictLogging {
     val chunkTable = getOrCreateChunkTable(dataset)
     val partIndexTable = getOrCreatePartitionIndexTable(dataset)
     clusterConnector.createKeyspace(chunkTable.keyspace)
-    val indexTable = getOrCreateIndexTable(dataset)
+    val ingestionTable = getOrCreateIngestionTimeIndexTable(dataset)
```

Comment:
I think `indexTable` is still the better variable name here.  It is still an index.  `ingestionTable` makes it sound like it is the primary table for ingestion data, which is not true.